common/tslint.yml

```diff
@@ -3,6 +3,16 @@ extends:
     - 'tslint:recommended'
 jsRules:
 rules:
+    await-promise: true
+    no-floating-promises: true
+    strict-boolean-expressions:
+      - true
+      - allow-mix
+      - allow-undefined-union
+      - allow-null-union
+      - allow-string
+      - allow-number
+      - ignore-rhs
```

Comment:
Cool!  (Also good to see TSLint invented a new way to write a very limited set of generics, https://palantir.github.io/tslint/rules/strict-boolean-expressions/)